### DISCUSSION TITLE
Ra-08(H) can be used?

### DISCUSSION THREAD

**Discussion by michapr** [jgromes/RadioLib#996] [Q&A]
Hi,

somebody have tried to use the Ai-Thinker device Ra-08 / Ra-08H with LoRaWan?
I have nothing found about it, even no information about the possible configuration (pins)...

Thanks!

**Comment by jgromes** [comment]
Quick google search shows the following document: https://docs.ai-thinker.com/_media/lora/docs/ra-08_v1.1.0_specification.pdf
On page 4 of that document, there's an overview of the device. It also states that this is an SoC (system-on-chip), which integrates the RF module alongside an ASR6601 microcontroller. I don't know if it will work with this library, although RadioLib does support CubeCell Arduino core, which seems to include ASR6601 support as well.

![Screenshot_79](https://github.com/jgromes/RadioLib/assets/3407192/53037dd5-04d4-478a-8430-68968c720d79)

**Comment by michapr** [comment]
@jgromes Sure, this I have found all... :)
Have also found that Cortex M4 and SX1262 inside...

But I nowhere have seen the internal connections from Cortex M4 to the SX1262 - this is not written in any document I have found (or I haven't seen it inside...)
So I have no idea how to configure the device in lib.

That's why was asking ;)

**Comment by michapr** [comment]
In the meantime I have found some information that the module is not Arduino-compatible. Too bad...

I was looking for an inexpensive solution for another good ready-made LoRaWan module.

**Reply by jgromes** [reply]
> I have found some information that the module is not Arduino-compatible

Where exactly? CubeCell claims to support his MCU, see here: https://github.com/HelTecAutomation/CubeCell-Arduino

And even if that core doesn't support it for some reason, RadioLib is not dependent on Arduino. It can be easily ported to non-Arduino environments, see https://github.com/jgromes/RadioLib/tree/master/examples/NonArduino

**Reply by michapr** [reply]
> Where exactly?

Have found in different posts searching (Google) for Ra-08 and Arduino. Also not sure why maybe not supported,... In my opinion there is no reason for this...
That's why was thinking about that somebody before already have tried and know the pin mappings ;)

>CubeCell claims to support his MCU

Yes, this is working fine, already have tested

**Comment by LoL-2003** [comment]
how we can use RA-08h for one-wire communication.

**Reply by HeadBoffin** [reply]
Very probably, very easy to try, but totally unrelated to RadioLib.
